Desktop/modules/dynamicmodules.cpp

```diff
@@ -111,21 +111,24 @@ bool DynamicModules::initializeModule(Modules::DynamicModule * module)
 		}
 
 		_modules[moduleName] = module;
-
+		
+		
 		if(!module->initialized())
-		{
+		{	
 			connect(module, &Modules::DynamicModule::registerForLoading,			this, &DynamicModules::registerForLoading			);
 			connect(module, &Modules::DynamicModule::registerForInstalling,			this, &DynamicModules::registerForInstalling		);
 			connect(module, &Modules::DynamicModule::registerForInstallingModPkg,	this, &DynamicModules::registerForInstallingModPkg	);
 			connect(module, &Modules::DynamicModule::descriptionReloaded,			this, &DynamicModules::descriptionReloaded			);
 
+			emit loadModuleTranslationFile(module);
+			
 			module->initialize();
 		}
 
 		if(!wasAddedAlready)
 		{
 			emit dynamicModuleAdded(module);
-			emit loadModuleTranslationFile(module);
+			//emit loadModuleTranslationFile(module);
```

Comment:
Why is it commented out? If it is really not necessary, just remove it.

Reply:
The emit was moved to the  ! initialized but was unnecessary. I've put it back

Reply:
So suppress this line of code. I don't like to see commented out codes when it's not explained

Reply:
Double that ;)